gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -3000,7 +3006,7 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     logical              :: do_myjsfc         = .false.               !< flag for MYJ surface layer scheme
     logical              :: do_myjpbl         = .false.               !< flag for MYJ PBL scheme
 #endif
-    integer              :: nmtvr          = 14                       !< number of topographic variables such as variance etc
+    integer              :: nmtvr          = 24                       !< number of topographic variables such as variance etc
```

Comment:
Isn't this the default value used by other GWD physics? If so, please do not change, just make sure to set it correctly in `input.nml` or adjust it in `GFS_typedefs.F90` if the drag suite is used.